README.md

```
@@ -151,53 +159,57 @@ Create a `.env` file in the root of your project with variables specified in the
+bash curl -X POST [http://localhost:8000/auth/register](http://localhost:8000/auth/register)
+-H "Content-Type: application/json"
+-d '{ "email": "user@example.com", "password": "securepassword" }'
+``` 
 
 ### Login
 
-```bash
-curl -X POST http://localhost:<PORT>/auth/login \
--H "Content-Type: application/json" \
--d '{ "email": "user@example.com", "password": "securepassword" }'
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)